apl-core/pom.xml

```diff
@@ -292,6 +292,14 @@
                     </filesets>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
```

Comment:
I suppose that should be configured in 'top level' POM file for all sub-modules
`<source>11</source>`